### PR TITLE
Verification and identity endpoints

### PR DESCRIPTION
**Pull request by brettmortensen**
Adds 5 endpoints for verification and identity. (verify, account numbers by account guid, account numbers by member guid, identify, and account owners)

@film42

### CODE REVIEW

File: client/account_number.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	}
+
+	buffer := new(bytes.Buffer)
+	buffer.ReadFrom(response.Body)
+	bufferStr := buffer.String()
+	response.Body.Close()
+
+	if response.StatusCode == 200 {
+		accountNumbersResponse := &models.AccountNumbersResponse{}
+		json.Unmarshal([]byte(bufferStr), accountNumbersResponse)
```
Comment (film42) [inline client/account_number.go]: `json.Unmarshal` returns an `error`, so you're going to need to check that and return if not `nil`. Same for a few other places in this PR.

**Review by film42 — CHANGES_REQUESTED** [review]

File: client/account_number.go (hunk 1 above)
Comment (film42) [reply client/account_number.go]: Otherwise, LGTM

Comment (brettmortensen) [reply client/account_number.go]: Good catch. Error handling has been added.

**Review by film42 — APPROVED** [review]
Looks good!

**Pull request merged**